### PR TITLE
RTRD over S3: parallel delete support

### PR DESCRIPTION
**Pull request by SergeyKaydalov**
#### Submitter checklist

- [ ] Change is code complete and matches issue description.
- [ ] Change is covered by existing or new tests.
- [ ] Change modifies documentation or comments only.

#### Which issue this PR fixes

fixes #

### CODE REVIEW

File: src/ccow/src/libccowfsio/fsio_inode.c [hunk 1]
```diff
@@ -259,23 +259,19 @@ __inode_free(ccowfs_inode * inode)
 	if (inode == NULL)
 		return;
 	log_trace(fsio_lg, "inode: %lu (%p)", inode->ino, inode);
-
+	log_debug(fsio_lg, "syncing inode %ju, before free", inode->ino);
```
Comment (SergeyKaydalov) [inline src/ccow/src/libccowfsio/fsio_inode.c]: @rayddteam Did you mean to move ccowfs_inode_sync(inode, 0) call under the line 271?

Comment (rayddteam) [reply src/ccow/src/libccowfsio/fsio_inode.c]: No, I just overlooked removed part in "raw patch" view. Sorry for noise.

Comment (SergeyKaydalov) [reply src/ccow/src/libccowfsio/fsio_inode.c]: closing then

**Pull request merged**